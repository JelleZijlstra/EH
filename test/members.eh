#!/usr/bin/ehi
printvar Object.members()
