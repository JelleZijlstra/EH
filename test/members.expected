%%stdout%%
@array [
0 => @string 'compare'
1 => @string 'finalize'
2 => @string 'inherit'
3 => @string 'initialize'
4 => @string 'isA'
5 => @string 'members'
6 => @string 'new'
7 => @string 'operator!='
8 => @string 'operator<'
9 => @string 'operator<='
10 => @string 'operator<=>'
11 => @string 'operator=='
12 => @string 'operator>'
13 => @string 'operator>='
14 => @string 'toString'
15 => @string 'type'
16 => @string 'typeId'
]
%%stderr%%
